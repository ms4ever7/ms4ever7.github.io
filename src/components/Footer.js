import React from 'react';

const Header = () => <footer className='footer layout-column layout-align-center-center'>
  <div className='footer-title'>&copy; Designed and developed by ms4ever7 </div>
</footer>;

export default Header;
