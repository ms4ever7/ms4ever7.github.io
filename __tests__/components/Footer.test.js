import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Footer from '../../src/components/Footer';

describe('Footer', () => {
  let wrapper;


  it('should render correctly', () => {
    wrapper = shallow(
      <Footer />
    );

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('has appropriate classnames', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper.find('.footer').exists()).toBeTruthy();
    expect(wrapper.find('.footer-title').exists()).toBeTruthy();
  });
});
