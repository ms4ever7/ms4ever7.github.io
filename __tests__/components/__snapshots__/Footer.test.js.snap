// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer
  className="footer layout-column layout-align-center-center"
>
  <div
    className="footer-title"
  >
    Â© Designed and developed by ms4ever7 
  </div>
</footer>
`;
